CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ file is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and
 this project adheres to [Semantic
 Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+
+- Added `update_config` request handler.
+
+### Removed
+
+- Removed `reload_config` request handler.
```

Comment:
I think these should be combined into one item in the `Changed` section.

Reply:
Applied the suggested adjustment.